prow/plugins/trigger/ic.go

```diff
@@ -26,7 +26,7 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-var okToTest = regexp.MustCompile(`(?m)^(@k8s-bot )?ok to test\r?$`)
+var okToTest = regexp.MustCompile(`/ok-to-test`)
```

Comment:
noted! i thought we only need to have `/test` to work alongside with `@k8s-bot test this` for a few weeks. it makes sense to have all bot comments doing so. i will also add test cases for `/joke`.